### ISSUE TITLE
Keyboard focus on column chart with points above `pointDescriptionEnabledThreshold` results in errors

### ISSUE THREAD

**Issue by khiga8** [highcharts/highcharts#22199]
#### Expected behaviour

Keyboard navigation should not result in errors in the console.

#### Actual behaviour

By default, Highcharts stops exposing datapoints to screen reader users when a chart renders more than 200 points (related: [pointDescriptionEnabledThreshold](https://api.highcharts.com/highcharts/accessibility.series.descriptionFormatter)).

I noticed that when I render a column chart that renders points above the `pointDescriptionEnabledThreshold` threshold, there are a bunch of errors on the console

```
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: path
Ancestor with aria-hidden:  <path fill=​"#544fc5" d=​"M 50.5 492.5 L 53.5 492.5 A 0 0 0 0 1 53.5 492.5 L 53.5 507 A 1.5 1.5 0 0 1 52 508.5 L 52 508.5 A 1.5 1.5 0 0 1 50.5 507 L 50.5 492.5 A 0 0 0 0 1 50.5 492.5 Z" stroke=​"#ffffff" stroke-width=​"1" opacity=​"1" filter=​"none" class=​"highcharts-point highcharts-negative highcharts-color-1" tabindex=​"-1" aria-hidden=​"true" style=​"outline:​ none;​">​</path>​
```

I don't see these errors with other types of Charts like https://jsfiddle.net/bxmwup2c/latest/.

There appears to be a difference in how `pointDescriptionEnabledThreshold` is handled across charts. Some charts stop exposing points to screen reader users by not rendering `<path>` elements at all. Others will continue rendering `<path>` elements but simply set `aria-hidden` on them. Setting `aria-hidden` on a focusable element is problematic accessibility wise.

#### Live demo with steps to reproduce

Chart with errors - https://jsfiddle.net/ezfo3cub/latest/
Chart with no errors - https://jsfiddle.net/bxmwup2c/latest/

#### Product version

Highcharts 11.4.8

#### Affected browser(s)

Chrome

**Comment by oysteinmoseng** [comment]
Actually this error/warning is accurate - as you have disabled screen reader description for the points, but not keyboard navigation. In other words, a user could still navigate to the points using their keyboard, but would not get proper descriptions if they are using a screen reader.

In case of the patch/workaround in #22149 this is deliberate, since you handle screen reader announcements separately. In other words the error could be ignored, or you could manually remove the `aria-hidden`s from each `point.graphic.element`.

Otherwise, the option [pointNavigationEnabledThreshold](https://api.highcharts.com/highcharts/accessibility.keyboardNavigation.seriesNavigation.pointNavigationEnabledThreshold) should be used to control whether or not the points can be focused.

We'll update the docs to better reflect this for others that encounter similar issues.
